JanePhotoBrowser/JanePhotoBrowser/Classes/PhotoBrowserView.swift

```diff
@@ -9,8 +9,11 @@ import UIKit
 
 @IBDesignable
 public class PhotoBrowserView: UIView {
+    
     //MARK: - Private Variables
-    fileprivate let collectionView: UICollectionView = UICollectionView(frame: CGRect.zero, collectionViewLayout: PhotoBrowserView.layout())
+    
+    fileprivate var largeImagesCollectionView: UICollectionView = UICollectionView(frame: CGRect.zero, collectionViewLayout: PhotoBrowserView.largeImagesLayout())
+    fileprivate var smallImagesCollectionView: UICollectionView = UICollectionView(frame: CGRect.zero, collectionViewLayout: PhotoBrowserView.smallImagesLayout())
```

Comment:
Make this guy nil and then make a flag where the user can set `showPreviews` to true if they want this functionality (off by default to make it backwards compatible). Then everywhere in this file make it handle nil properly.